core/src/main/scala/com/evolutiongaming/catshelper/CatsHelper.scala

```diff
@@ -146,21 +147,34 @@ object CatsHelper {
     }
   }
 
-  implicit final class ResourceLogOps[F[_], A](val self: Resource[F, A]) extends AnyVal {
-    def log(name: String)(implicit F: Sync[F], log: Log[F], md: MeasureDuration[F]): Resource[F, A] = Resource {
+  implicit final class ResourceObservabilityOps[F[_], A](val self: Resource[F, A]) extends AnyVal {
```

Comment:
this is not backward compatible change :(

Reply:
I think this is just how diff is displayed. I didn't rename ResourceLogOps, ResourceLogOps is still there.